### PR TITLE
Making a pull request for my changes to talk more with the child

### PR DESCRIPTION
**Pull request by bensky**
Please review this, @casey-collab

### CODE REVIEW

File: english/explore-outside/talk-child/talk-more/talk-more.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+As you continue to walk closer to the child you see that his red eys are from crying.
+He actually looks very sad.  
+
+He says in a thick Pittsburgh accent, "Why did yuns eat the Marshmellow house? That's my house!"
+
+What do you say back?
+
+[Oh sorry, I was trapped in there. How do I get home?](./nice-talk/nice-talk.md)
```
Comment (casey-collab) [inline english/explore-outside/talk-child/talk-more/talk-more.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/explore-outside/talk-child/talk-more/talk-more.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+[You have some nerve, trapping me in there! Give me your boat!!](./indignant-talk/indignant-talk.md)
```
Comment (casey-collab) [inline english/explore-outside/talk-child/talk-more/talk-more.md]: This link is also invalid.

File: english/explore-outside/talk-child/talk-more/talk-more.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+[Tell me how to get to the tower or I'll eat your boat next.](./threaten-talk/threaten-talk.md)
```
Comment (casey-collab) [inline english/explore-outside/talk-child/talk-more/talk-more.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by bensky** [comment]
Please review this, @casey-collab

File: english/explore-outside/talk-child/talk-more/nice_talk/nice-talk.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+The child says, "Oh OK, so sorry for trapping you. Must have been a misunderstanding."
```
Comment (casey-collab) [inline english/explore-outside/talk-child/talk-more/nice_talk/nice-talk.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by bensky** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cherry_blossom:

**Pull request merged**